cypress/support/step_definitions/index.ts

```diff
@@ -91,7 +91,9 @@ Then(/^the "(.*)" element[s]? exist[s]? ([0-9]*) time[s]?$/, async (selector: st
 Then(/^the "(.*)" element(s)? contain[s]?$/, async (selector: string, multiple = '', table: DataTable) => {
   const rows = table.rows()
   if (multiple === 's') {
-    $(selector).each((el, i) => {
+    const tableHeads = $(selector).should('have.length', rows.length)
+
+    tableHeads.each((el, i) => {
```

Comment:
Oh sorry I meant use this existing step:

https://github.com/kumahq/kuma-gui/blob/93a255521efe545f50aadd97177d7ec9bfc2b5f6/features/mesh/gateways/Index.feature#L45

If we add this code you've added here it means I can't just check the first few elements of a selector, i.e. it does too much, plus we already have a step that tests length / times occurring.

Reply:
I think now I got it right